app/src/main/java/com/cornellappdev/transit/ui/components/MenuItem.kt

```diff
@@ -24,16 +28,17 @@ import androidx.compose.ui.unit.sp
  * @param sublabel The sublabel for the item
  */
 @Composable
-fun MenuItem(icon: ImageVector, label: String, sublabel: String) {
+fun MenuItem(icon: ImageVector, label: String, sublabel: String, onClick: () -> Unit) {
     Row(Modifier.fillMaxWidth(), verticalAlignment = Alignment.CenterVertically) {
         Icon(
             icon,
             contentDescription = "Place",
-            modifier = Modifier.padding(horizontal = 10.dp)
+            modifier = Modifier.padding(end=20.dp),
+            tint = TransitBlue
         )
         Column() {
-            Text(text = label, fontSize = 24.sp)
-            Text(text = sublabel, fontSize = 16.sp)
+            Text(text = label, fontSize = 14.sp, color = PrimaryText, fontFamily = sfProDisplayFamily)
+            Text(text = sublabel, fontSize = 10.sp, color = SecondaryText, fontFamily = sfProDisplayFamily)
```

Comment:
Lint the file :-)

![image](https://github.com/cuappdev/ithaca-transit-compose/assets/47724806/f11a04c3-9c0e-43e0-8ff7-37c071cdb629)